content/doc/book/security/securing-jenkins.adoc

```diff
@@ -58,7 +58,7 @@ users of Jenkins from indirect attacks.
 The following topics discuss features that are *off by default*.
 We recommend you read them first and act on them immediately.
 
-* https://wiki.jenkins.io/display/JENKINS/CSRF+Protection[CSRF Protection] --- prevent a remote attack against Jenkins running inside your firewall. This feature is *off by default* in Jenkins 1.x and when upgrading to 2.x.
+* link:/doc/book/security/csrf-protection[CSRF Protection] --- prevent a remote attack against Jenkins running inside your firewall.
```

Comment:
following topics discuss features off by default? this doesn't seem to be in the right place.

Reply:
Good point, I didn't look at the context. Will fix.

Reply:
Also moved Agent-to-Controller to the 'on by default' list. No idea when that happened, but it did.